web-registry/src/pages/CreditClassDetails/CreditClassDetails.utils.ts

```diff
@@ -39,3 +51,42 @@ export const getCreditClassAvgPricePerTonLabel = ({
 
   return avgPricePerTonLabel;
 };
+
+/* parseCreditClassVersion  */
+
+export const parseCreditClassVersion = (
```

Comment:
could we re-use this too in `parseOffChainProject` from `ProjectTopSection.utils.ts`?